### PR TITLE
Build screenshot layer for MacOS

### PR DESCRIPTION
**Pull request by juan-lunarg**
Tested it locally using vkconfig and it worked just like on Linux.

### CODE REVIEW

**Comment by juan-lunarg** [comment]
Pointed to my local version of vkcube and it worked without issue.

```

Launching Vulkan Application:
- Layers overridden by "Screenshot" configuration.
- Application: my_vkcube
- Executable: /Users/juanramos/projects/KHR-Vulkan-Tools/build/cube/vkcube.app/Contents/MacOS/vkcube
- Working Directory: /Users/juanramos/projects/KHR-Vulkan-Tools/build/cube/vkcube.app/Contents/MacOS
- Command-line Arguments: --suppress_popups
- Log file: /Users/juanramos/VulkanSDK/vkcube.txt

screenshot: Selected format:USE_SWAPCHAIN_COLORSPACE
Is NOT in the list:
UNORM, SNORM, USCALED, SSCALED, UINT, SINT, SRGB
Swapchain Colorspace will be used instead

Selected GPU 0: Apple M2 Max, type: IntegratedGpu

screenshot: Capture file is: /Users/juanramos/VulkanSDK/1.ppm 

Process terminated
```

**Review by charles-lunarg — APPROVED** [review]
Nice, just a 'make it compile' change with no really custom bits needed.

**Comment by juan-lunarg** [comment]
> Nice, just a 'make it compile' change with no really custom bits needed.

Yeah when I was working on cleaning up the layer installation code. I got curious and was wondering what it would take to make this work.

Turns out almost nothing.

File: layersvt/screenshot.cpp [hunk 1]
```diff
@@ -51,53 +51,6 @@ const char *kSettingsKeyFrames = "frames";
 const char *kSettingKeyFormat = "format";
 const char *kSettingKeyDir = "dir";
 
-#ifdef ANDROID
```
Comment (richard-lunarg) [inline layersvt/screenshot.cpp:54]: Why is all this being removed?

Comment (juan-lunarg) [reply layersvt/screenshot.cpp:54]: It does nothing. Should have removed when we integrated the utility libraries which handles this logic now.

Comment (richard-lunarg) [reply layersvt/screenshot.cpp:54]: Ah, great. It's the only thing that didn't make sense "from the outside".

**Review by richard-lunarg — APPROVED** [review]
Great work! Just building it for macOS is a testament to Vulkan's portability. Thanks also for adding the portability extension to the tests, this often get's missed.

**Pull request merged**